<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1263</width>
      <height>627</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: #CBF7ED;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="boton_adelante">
     <property name="geometry">
      <rect>
       <x>1120</x>
       <y>170</y>
       <width>93</width>
       <height>297</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>93</width>
       <height>297</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/resources/resources/right.png);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_atras">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>93</width>
       <height>297</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>93</width>
       <height>297</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-image: url(:/resources/resources/left.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>511</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #0F5257;
font: 87 30pt &quot;Segoe UI Black&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Seleccionar usuario:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>500</y>
       <width>271</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#E6FAFC;
font: 87 16pt &quot;Segoe UI Black&quot;;
background-color: #0B3142;</string>
     </property>
     <property name="text">
      <string>crear</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>981</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="usersLayout"/>
    </widget>
    <widget class="QPlainTextEdit" name="plain_user">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>500</y>
       <width>331</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: #0F5257;
color: rgb(0, 0, 0);
font: 87 14pt &quot;Segoe UI Black&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1237</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
